.github/workflows/release-python.yml

```diff
@@ -1,12 +1,15 @@
 name: Release Python
 
 on:
-  push:
-    tags:
-      - 'python/v**'
+  release:
+    types: [published]
+  workflow_dispatch:
+    description: "Manually publish release"
+
 
 jobs:
   macos:
+    if: ${{ github.event_name == "workflow_dispatch" || startsWith(github.event.release.tag_name, "python/v") }}
```

Comment:
This condition makes sure the workflow is only ran when a release for the respective crate has been created. Without it, a release created for the library crate would trigger this workflow and vice-versa.
 
Unfortunately we can't set this condition at the workflow level, so some duplication is required. Open to feedback and ideas on how to resolve that.